tests/test_bgp_neighbor.rb

```diff
@@ -215,6 +215,7 @@ def test_bgpneighbor_set_get_log_neighbor_changes
       neighbor = RouterBgpNeighbor.new(@@asn, vrf, @@addr)
       check = [:enable, :disable, :inherit, 'enable', 'disable', 'inherit',
                neighbor.default_log_neighbor_changes]
+      end
```

Comment:
Why is this `end` here? I can't imagine that the test file still works after this?